plugins/PrivacyManager/templates/privacySettings.twig

```diff
@@ -1,10 +1,9 @@
 {% extends 'admin.twig' %}
 
-{% set title %}{{ 'PrivacyManager_TeaserHeadline'|translate }}{% endset %}
+{% set title %}Anonymize data{% endset %}
```

Comment:
Should this be translated?

Reply:
yes, I was meaning to add translation keys after the wording is defined but will add this one now 👍